amoro-mixed-format/amoro-mixed-format-trino/src/main/java/org/apache/amoro/trino/MixedFormatConfig.java

```diff
@@ -21,8 +21,8 @@
 import io.airlift.configuration.Config;
 import io.airlift.configuration.ConfigDescription;
 
-/** Arctic config */
-public class ArcticConfig {
+/** Mixed-format configurations */
+public class MixedFormatConfig {
```

Comment:
Maybe `AmoroConfig`? The same applies to the following configuration. I feel amoro.xxx would be better, because it actually represents the connector name. Users set these configurations according to connector.xxx, such as iceberg.xxx. Or, does this mean our connector name is mix-forma, if so, please ignore.

Reply:
Yes，the new connector name is mixed-format, and it is used for mixed-format only now.